utils/state.py

```diff
@@ -0,0 +1,64 @@
+import os
+
+from botocore.errorfactory import ClientError
+
+from utils.aws_api import AWSApi
+
+
+class State(object):
+    """
+    A state object to be used by stateful integrations.
+    A stateful integration is one that has to do each action only once,
+    and there is no source of truth to validate against.
+
+    Good example: email-sender should only send each email once
+    Bad example: openshift-resources' source of truth is the clusters
+
+    :param integration: name of calling integration
+    :param accounts: Graphql AWS accounts query results
+    :param settings: App Interface settings
+
+    :type integration: string
+    :type accounts: list
+    :type settings: dict
+    """
+    def __init__(self, integration, accounts, settings=None):
+        """Initiates S3 client from AWSApi."""
+        self.state_path = f"state/{integration}"
+        self.bucket = os.environ['APP_INTERFACE_STATE_BUCKET']
+        account = os.environ['APP_INTERFACE_STATE_BUCKET_ACCOUNT']
+        aws_api = AWSApi(1, accounts, settings=settings)
+        session = aws_api.get_session(account)
+
+        self.client = session.client('s3')
+
+    def exists(self, key):
+        """checks if a key exists in the state
+
+        Arguments:
+            key {string} -- key to check
```

Comment:
Please use the same docstring format everywhere. Ideally, the format you're using in the class docstring, which is the one described in our style guide.

Reply:
done for ocm.py as well!